ios/ExpensifyCash/Info.plist

```diff
@@ -48,6 +48,17 @@
 			</dict>
 		</dict>
 	</dict>
+    <key>CFBundleURLTypes</key>
```

Comment:
the spacing here looks off compared to the previous closing </dict> and the next <key> on line 62.